src/router/private/index.tsx

```diff
@@ -0,0 +1,17 @@
+import { Navigate } from 'react-router-dom'
+
+import { Routes } from 'routes'
+
+type Props = {
+  children: React.ReactElement
+}
+
+const ProtectedRoute: React.FC<Props> = ({ children }) => {
+  const isAuthenticated = false
+  if (isAuthenticated) {
+    return children
+  }
+  return <Navigate to={{ pathname: Routes.LoginUrl() }} replace={true} />
+}
```

Comment:
Można trochę uprościć + dać powietrza żeby nie było takiego ścisku między wierszami + import React

zazwyczaj jest tak, że jeżeli prop jakiegoś komponentu jest boolean to nie trzeba robić `replace={true}`, wystarczy przekazać `replace` jeżeli chcemy, żeby zawsze przekazywać `true`

```suggestion
import React from "react"

...

const ProtectedRoute: React.FC<Props> = ({ children }) => {
  const isAuthenticated = false
  
  if (isAuthenticated) return children
  
  return <Navigate to={Routes.LoginUrl()} replace />
}
```